vue/vite.config.ts

```diff
@@ -14,6 +19,26 @@ export default defineConfig(({ mode }) => {
     base: env.BASE_PATH,
     plugins: [
       vue(),
+      // @if auto-import
+      AutoImport({
+        // @if !router
```

Comment:
resolvers: [ElementPlusResolver()],
```suggestion
      AutoImport({
        resolvers: [ElementPlusResolver()],
        // @if !router
```

Reply:
如果按需引入Element Plus的话 这里也要加 参考 https://element-plus.gitee.io/en-US/guide/quickstart.html#on-demand-import

Reply:
应该是这个链接
https://github.com/antfu/unplugin-vue-components#importing-from-ui-libraries

@zxqzy 把这个链接以注释的形式添加进去，默认的 resolvers 用 ElementPlusResolver